test/unit_test/utils.py

```diff
@@ -31,29 +33,9 @@ def forward(self, x):
         return torch.relu(self.fc1(x))
 
 
-class DummyDataset(Dataset):
-    def __init__(self, size):
-        self.size = size
-
-    def __getitem__(self, idx):
-        return torch.randn(1), torch.rand(10)
-
-    def __len__(self):
-        return self.size
-
-
-class FixedDummyDataset(Dataset):
-    def __init__(self, size):
-        self.size = size
-        self.rng = np.random.default_rng(0)
-        self.data = torch.tensor(self.rng.random((size, 1)))
-        self.labels = torch.tensor(self.rng.random(1))
-
-    def __getitem__(self, idx):
-        return self.data[idx], self.labels[idx]
-
-    def __len__(self):
-        return self.size
+# TODO(shaahji): Remove this once perf_tuning pass supports DataConfig
+def create_dummy_dataloader(data_dir, batch_size=1, **kwargs):
```

Comment:
can you make the size of this dataset be `1`? Otherwise, the quantization_aware_training is taking a long time to run.

Reply:
the old dataloader also only had size 1

Reply:
Done!